tools/add_triage_signature.py

```diff
@@ -2037,6 +2037,44 @@ def _process_ticket(self, url, issue_key):
             self._update_triaging_ticket("\n".join(host_messages))
 
 
+class ErrorOnCleanupInstallDevice(Signature):
+    """
+    This signature monitors errors on cleanupInstallDevice function
+    """
+
+    # This has to be maintained to be the same format as
+    # https://github.com/openshift/assisted-installer/blob/51c021f3245ef1d9e1a50a3e31dc23350cdc5d32/src/installer/installer.go#L98
+    # Remember to maintain backwards compatibility if that format ever changes - create
+    # PATTERN_NEW and try to match both.
+    LOG_PATTERN = re.compile(r'msg="failed to prepare install device')
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(
+            *args,
+            **kwargs,
+            comment_identifying_string="h1. Error trying to perform cleanup on installation disk",
+        )
+
+    def _process_ticket(self, url, issue_key):
+        md = get_metadata_json(url)
+        cluster = md["cluster"]
+        cluster_id = cluster["id"]
+        triage_logs_tar = get_triage_logs_tar(triage_url=url, cluster_id=cluster_id)
+
+        for host in cluster["hosts"]:
+            host_id = host["id"]
+
+            try:
+                agent_logs = get_host_log_file(triage_logs_tar, host_id, "agent.logs")
```

Comment:
https://github.com/openshift/assisted-installer/pull/583 is modifying installer code, not agent code.

So need to look in `installer.logs` and not in `agent.logs`

Reply:
Oops ... fixed!